scripts/bash/lxd-client

```diff
@@ -94,7 +94,10 @@ _have lxc && {
       security.nesting security.privileged security.protection.delete \
       security.protection.shift security.syscalls.blacklist \
       security.syscalls.blacklist_compat security.syscalls.blacklist_default \
-      security.syscalls.intercept.mknod security.syscalls.intercept.setxattr \
+      security.syscalls.intercept.mknod security.syscalls.intercept.mount \
+      security.syscalls.intercept.mount.allowed \
+      security.syscalls.intercept.setxattr \
+      security.syscall.intercept.mount.shift \
```

Comment:
This should be in a separate `scripts: Add security.syscalls.intercept.mount to bash completion` commit